application/templates/cms/search_data_source.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% block pageTitle %}Search for an existing data source{% endblock %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for('static_site.topic', topic_slug=topic.slug), "text": topic.title}
+  ]
+%}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form method="GET" action="{{ url_for('cms.search_data_sources', topic_slug=topic.slug, subtopic_slug=subtopic.slug, measure_slug=measure.slug, version=measure_version.version )}}">
+
+        <div class="govuk-form-group">
+
+          <span class="govuk-caption-xl">{{ measure_version.title }} </span>
+          <h1 class="govuk-label-wrapper">
+            <label class="govuk-label govuk-label--xl" for="q">Search for an existing data source</label>
+          </h1>
+
+          <input class="govuk-input govuk-input--width-30" id="q" name="q" type="text" value="{{ q }}">
+        </div>
+
+
+        <button class="govuk-button">Search</button>
+
+      </form>
+
+      {% if not data_sources == None %}
+
+        {% if data_sources|length > 0 %}
+
+          {% if data_sources|length == 100 %}
+
+            <p class="govuk-body">Only the first 100 results shown below.</p>
```

Comment:
Can we use a warning text for this, rather than plain text? It doesn't stand out very much.

I might be tempted to put it after the results, as well - but I can see pros/cons to either way.

https://design-system.service.gov.uk/components/warning-text/

Reply:
@samuelhwilliams warning text should be reserved for much more serious warnings, I think. I did consider [Inset text](https://design-system.service.gov.uk/components/inset-text/) but sometimes that can actually decrease people's likelihood of reading it.

I suspect that in this instance, plain text is probably fine. It also doesn't matter hugely if people miss the text as you should soon realise that there’s a lot of results shown by scrolling down, and I suspect you'd be more likely to refine the search rather than check them all. Also hopefully there will soon be less than 100 in most searches anyway, as we de-duplicate.

Reply:
Ok, I'll take that on warning text - it's not super serious.

It doesn't look very good as just plain text though - it gets completely lost. It feels like it needs to stand out at least a little bit.

Reply:
Let's test it and see? We can increase its prominence later if it proves problematic.